3-typescript/2-custom-utility-types-and-generics/1-deep-clone.ts

```diff
@@ -12,3 +12,35 @@
  */
 
 //? implement the function  here
+import { readJsonFile } from "../1-ecommerce/utils/read-json.util";
+
+import {  Brand } from "../1-ecommerce/1-types";
+
+const deepClone = (obj: any): typeof obj => {
```

Comment:
We we're expecting more a generic method here 
```suggestion
const deepClone = <T>(obj: T): T => {
```